cdap-ui/app/cdap/components/PipelineScheduler/BasicView/Summary.js

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import moment from 'moment';
+import {INTERVAL_OPTIONS} from 'components/PipelineScheduler/Store';
+
+const mapStateToSummaryProps = (state) => {
+  return {
+    state
+  };
+};
+
+const mapStateToMaxConcurrentRunsSummaryProps = (state) => {
+  return {
+    maxConcurrentRuns: state.maxConcurrentRuns
+  };
+};
+
+const SummaryComponent = ({state}) => {
+  switch (state.intervalOption) {
+    case INTERVAL_OPTIONS['5MIN']:
+      return <span>every 5 minutes</span>;
+    case INTERVAL_OPTIONS['10MIN']:
+      return <span>every 10 minutes</span>;
+    case INTERVAL_OPTIONS['30MIN']:
+      return <span>every 30 minutes</span>;
+    case INTERVAL_OPTIONS['HOURLY']:
```

Comment:
Why not just `case INTERVAL_OPTIONS.HOURLY`?

Reply:
Went ahead and changed the `HOURLY`, `MONTHLY` etc. ones, but couldn't change the minutes ones e.g `5MIN` because Javascript doesn't allow it